ui-kit/src/components/Column/index.tsx

```diff
@@ -0,0 +1,62 @@
+import classNames from 'classnames';
+import React, { HTMLAttributes } from 'react';
+import '../../sass/components/_Column.scss';
+
+type NumberAttribute =
+  | number
+  | '1'
+  | '2'
+  | '3'
+  | '4'
+  | '5'
+  | '6'
+  | '7'
+  | '8'
+  | '9'
+  | '10'
+  | '11'
+  | '12';
```

Comment:
지금 상태라면 `NumberAttribute`에 아무 숫자나 다 넣을 수 있으니

```suggestion
type NumberAttribute = 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12;
```

요렇게 그냥 타입을 `number`로 통일해서 요렇게 해도 되지 않을까 합니다...!

Reply:
수정했습니당